README.md

```diff
@@ -39,3 +39,35 @@ Every query returns a JSON object with a `TYPE` attribute which is either `cours
 
 After you get an app working with the dummy data from `get_info.php`, you can tackle getting a mariadb instance
 set up and having a php script gather information from a database.
+
+## Database Setup
+Login to database via Mariadb (I used MYSQL client via windows)
+
+Once logged in, follow the tutorial on the following website to setup database and assign user to it
+http://www.daniloaz.com/en/how-to-create-a-user-in-mysql-mariadb-and-grant-permissions-on-a-specific-database/
```

Comment:
URLs disappear all the time. Go ahead and repeat the relevant instructions here.